defusedxml/ElementTree.py

```diff
@@ -61,11 +61,11 @@ def _get_py3_cls():
 
 class DefusedXMLParser(_XMLParser):
 
-    def __init__(self, html=0, target=None, encoding=None,
+    def __init__(self, target=None, encoding=None,
```

Comment:
Removing the `html` argument lose the compatibility with ElementTree library interface (also with Python 2.7, that this library supports, where the argument is not deprecated).
May be better leaving it but ignore it?

Reply:
In that case, I think it's better to add proper compatibility by using `sys.version_info` to expose a different signature depending on the version.

Reply:
Signature fixed in 1a295405fd589cf2ee2b7809de6d39454c0d6235